applications/nrf_desktop/doc/usb_state.rst

```diff
@@ -6,3 +6,67 @@ USB state module
 The ``usb_state`` module is responsible for tracking the USB connection.
 It is also responsible for the data transmission that application does on this
 device.
+
+Module Events
+*************
+
++--------------------+----------------------------+-------------+-----------------------------------+------------------+
+| Source Module      | Input Event                | This Module | Output Event                      | Sink Module      |
++====================+============================+=============+===================================+==================+
++                    + ``module_state_event``     +             + ``module_state_event``            +                  +
++--------------------+----------------------------+             +-----------------------------------+------------------+
++ ``hid_state``      + ``hid_report_event``       +             + ``hid_report_sent``               + ``hid_state``    +
++--------------------+----------------------------+             +-----------------------------------+                  +
++                    + ``config_event``           +             + ``hid_report_subscription_event`` +                  +
++--------------------+----------------------------+             +-----------------------------------+------------------+
++                    + ``config_fetch_event``     +             + ``usb_state_event``               +                  +
++--------------------+----------------------------+             +-----------------------------------+------------------+
++                    + ``config_forwarder_event`` +             +                                   +                  +
++--------------------+----------------------------+-------------+-----------------------------------+------------------+
```

Comment:
Same comment as I made in: https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/2103#discussion_r406051650
Personally, I don't understand the meaning of this table without looking into the source code.

I think this could be improved to our reader, maybe a diagram, and additional text to help the reader.

Reply:
I discussed this with the nRF Desktop team yesterday. We're thinking about adding an explanation for each column on the main application page, where all modules are listed, and link to that section from each module page.